databases/week3/dbweek3.sql

```diff
@@ -0,0 +1,261 @@
+);
+
+-- Inserts to meal table
+INSERT INTO 
+  meal(title, description, location, price, created_date)
+VALUES
+  ('Carbonara', 'Designating pasta that is mixed, just after boiling, with a sauce of sautéed bacon, 
+  Parmesan cheese, pepper, and beaten eggs, formed as the eggs are cooked slightly by the hot pasta.', ' Holsteinsgade 1, 8300 Odder',
+   144.50, '2021-10-10');
+
+INSERT INTO 
+  meal(title, description, location, price, created_date)
+VALUES
+  ('Avocado toast', 'Avocado toast can be a nutritious breakfast, as avocados are a good source of healthy fats and very filling.', ' Holsteinsgade 1, 8660 Skanderborg',
+   70.00, '2021-10-13');
+
+INSERT INTO 
+  meal(title, description, location, price, created_date)
+VALUES
+  ('Sticky and Spicy Baked Cauliflower', 'Battering and oven-baking large cauliflower florets gives them a slightly crackly, just-rich-enough exterior that’s ideal for coating in a sticky-sweet gochujang glaze.', ' Holsteinsgade 1, 8660 Skanderborg',
+   170.40, '2021-10-14');
+
+INSERT INTO 
+  meal(title, description, location, price, created_date, max_reservations)
+VALUES
+  ('Greek salat', 'Popular salad generally made with pieces of tomatoes, cucumbers, onion, feta cheese, and olives and dressed with salt, pepper, Greek oregano, and olive oil.', ' M. P. Bruuns Gade 25, 8000 Aarhus',
+   100.10, '2021-10-11', 3);
+
+INSERT INTO 
+  meal(title, description, location, price, created_date, max_reservations)
+VALUES
+  ('Vanilla Cake', 'A classic butter cake but with Japanese techniques applied for the most plush, soft and moist yellow cake like you’ve never had before.', ' M. P. Bruuns Gade 25, 8000 Aarhus',
+   60.50, '2021-09-11', 5);
+
+-- Inserts to reservation table
+INSERT INTO
+  reservation(number_of_guests, created_date, contact_phonenumber, contact_name, meal_id)
+VALUES(3, '2021-10-11', '+ 45 50 000 00', 'John', 1);
+
+INSERT INTO
+  reservation(number_of_guests, created_date, contact_phonenumber, contact_name, meal_id)
+VALUES(5, '2021-11-11', '+ 45 50 001 00', 'Alex', 7);
+
+INSERT INTO
+  reservation(number_of_guests, created_date, contact_phonenumber, contact_name, meal_id)
+VALUES(2, '2021-11-12', '+ 45 50 001 10', 'Maria', 5);
+
+INSERT INTO
+  reservation(number_of_guests, created_date, contact_phonenumber, contact_name, meal_id)
+VALUES(4, '2021-11-13', '+ 45 50 001 11', 'Natalia', 6);
+
+-- Inserts to review table
+INSERT INTO
+  review(title, review_meal_id, created_date, stars)
+VALUES('Review for carbonara', 1 , '2021-08-12', 3 );
+INSERT INTO
+  review(title, review_meal_id, created_date, stars)
+VALUES('Review avocado toast', 5 , '2021-09-16', 7 );
+INSERT INTO
+  review(title, review_meal_id, created_date, stars)
+VALUES('Review for Greek salat', 7 , '2021-10-12', 9 );
+INSERT INTO
+  review(title, review_meal_id, created_date, stars)
+VALUES('Review for vanill cake', 8 , '2021-08-19', 1 );
+
+-- Meal - Queries to write
+-- Get all meals
+SELECT 
+  title
```

Comment:
The task asked for meals not meals title, so in this case make sense to bring all columns on the select.

Reply:
this goes for all queries that don't specify columns